db/migrate/20090714132449_add_audits_table.rb

```diff
@@ -12,6 +12,7 @@ def self.up
       t.column :comment, :string
       t.column :auditable_parent_id, :integer
       t.column :auditable_parent_type, :string
+      t.column :request_uuid, :string
```

Comment:
> given we add the column in separate migration, why to edit this old file as well

IIRC, updating to a new version of audits requires this column, else the audit gem fails when it can't find this column in the table.  It's probably indicating that we have buggy migrations which aren't disabling audits...